guides/04_Quality_Bandwidth_Connectivity.md

```diff
@@ -55,6 +55,23 @@ The Amazon Chime SDK for JavaScript produces several kinds of events on the [Aud
 |------------ | ------------- | ------------- |
 |[metricsDidReceive](https://github.com/aws/amazon-chime-sdk-js/blob/bf6d01e236445684601e24f3e319dede728b5113/src/audiovideoobserver/AudioVideoObserver.ts#L76) | Exposes the WebRTC getStats metrics. There may be differences among browsers as to which metrics are reported. | All |
 
+> Note: We have built a video WebRTC statistics widget in the demo browser meeting app. This widget is shown as an overlay on the video, when the video is enabled.
```

Comment:
I would mention here that this is currently only for Chrome.

Reply:
Yeah thanks for the catch, updated it to call it out explicitly.